airbyte-integrations/connectors/source-surveymonkey/source_surveymonkey/schemas/survey_questions.json

```diff
@@ -21,16 +21,70 @@
       "type": "string"
     },
     "layout": {
-      "type": ["object", "null"]
+      "type": [
+        "object",
+        "null"
+      ]
     },
     "sorting": {
-      "type": ["string", "null"]
+      "type": "object",
```

Comment:
it has to be `"type": ["object", "null"]`
because for airbyte testing accout we have 'null' values for this field